README.md

```diff
@@ -455,6 +503,14 @@ http://path-to-stream/live/channel-z.ts
   - `catchup-source`: Can contain the full catchup URL (complete with format specifiers) if in `default` mode. Or if the mode is `append` just the query string with format specifiers which will be appended to the channel URL. If omitted the `Query format string` from settings will be appended.
   - `catchup-days`: The number of days in the past catchup is available for.
   - `catchup-correction`: A correction to the time used for catchup stream URL generation. Useful for catchup streams which are geo mis-matched to the wrong time.
+  - `provider`: The name that idenitifes this provider and will be seen in the provider manager. Also that name that can be used for any provider mappings if required. No other provider are valid if this tag is omitted.
+  - `provider-type`: A type which must match one of addon, satellite, cable, aerial, iptv or unknown.
+  - `provider-logo`: A path to the location where the icon for this provider is available.
+  - `provider-countries`: The country for this provider. Should be passed using an ISO-3166 country code. eave empty or omit for unspecified.
```

Comment:
I'd make "ISO-3166" a link pointing to table of country codes or just use example in braces, just like `provider-languages` below. Also typo: s/eave/leave/